source/administration/config-in-database.rst

```diff
@@ -1,76 +1,78 @@
+
```

Comment:
Not sure if it's because of the added line break here or something else, but if I go to View File, the preview for this page looks off. I'm not 100% sure what's causing it

Reply:
I removed the extra line but it didn't fix the issue.